src/openrct2-ui/windows/ShortcutKeys.cpp

```diff
@@ -48,18 +48,25 @@ static rct_widget window_shortcut_widgets[] = {
 
 static constexpr const rct_string_id CHANGE_WINDOW_TITLE = STR_SHORTCUT_CHANGE_TITLE;
 static constexpr const int32_t CHANGE_WW = 250;
-static constexpr const int32_t CHANGE_WH = 60;
+static constexpr const int32_t CHANGE_WH = 80;
+
+enum
+{
+    WIDX_REMOVE = 3
+};
 
 // clang-format off
 static rct_widget window_shortcut_change_widgets[] = {
     WINDOW_SHIM(CHANGE_WINDOW_TITLE, CHANGE_WW, CHANGE_WH),
+    MakeWidget({ 75, 56 }, { 100, 12 }, WindowWidgetType::Button, WindowColour::Primary, STR_SHORTCUT_REMOVE, STR_SHORTCUT_REMOVE_TIP),
```

Comment:
12px is too small for CJK; suggest 14px instead.
```suggestion
    MakeWidget({ 75, 56 }, { 100, 14 }, WindowWidgetType::Button, WindowColour::Primary, STR_SHORTCUT_REMOVE, STR_SHORTCUT_REMOVE_TIP),
```